code/datums/abnormality/_ego_datum/he.dm

```diff
@@ -220,3 +220,12 @@
 /datum/ego_datum/weapon/songmini
 	item_path = /obj/item/gun/ego_gun/pistol/songmini
 	cost = 35
+
+//You Must Become Strong - GET POWERFUL
```

Comment:
Abno name - EGO name
```suggestion
//You Must Become Strong - Get Strong
```

Reply:
Could also just call it screwloose - the name of the E.G.O. in limbus